presentation-api/receiving-ua/support/PresentationConnection_terminate_receiving-ua.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+
+<meta charset="utf-8">
+<title>Terminating a presentation in a receiving browsing context</title>
+<link rel="author" title="Tomoyuki Shimizu" href="https://github.com/tomoyukilabs/">
+<link rel="help" href="https://w3c.github.io/presentation-api/#terminating-a-presentation-in-a-receiving-browsing-context">
+<script src="../common.js"></script>
+<script src="stash.js"></script>
+
+<script>
+  const stash = new Stash(stashIds.toReceiver, stashIds.toController);
+
+  navigator.presentation.receiver.connectionList.then(list => {
+    const connection = list.connections[0];
+
+    // terminate a presentation when two presentation connections become "connected"
+    list.onconnectionavailable = evt => {
+      connection.terminate();
+
+      // the following code must not be invoked since the presentation is terminated
+      stash.send({ type: 'error' });
```

Comment:
Right. I'll change the code into the following:

```js
stash.send({ type: (window.closed ? 'ok' : 'error') }); 
```

FYI: As far as I have tested on Firefox Nightly for Android, `window.closed` is not set to `true` yet as soon as calling "terminate",  and it becomes `true` when an `unload` event is fired. Anyway, I will fix the codes as you mentioned above.